_posts/en/newsletters/2020-07-01-newsletter.md

```diff
@@ -0,0 +1,371 @@
+---
+title: 'Bitcoin Optech Newsletter #104'
+permalink: /en/newsletters/2020/07/01/
+name: 2020-07-01-newsletter
+slug: 2020-07-01-newsletter
+type: newsletter
+layout: newsletter
+lang: en
+---
+This week's newsletter summarizes a discussion about mining incentives
+related to HTLCs and links to an announcement about a proposed service to
+store and relay presigned transactions.  Also included are our regular
+sections with recently transcribed talks and conversations, new releases
+and release candidates, and notable changes to popular Bitcoin
+infrastructure projects.
+
+## Action items
+
+*None this week.*
+
+## News
+
+{% comment %}<!-- FIXME Reviewers: the following item links to two of my
+posts; please be extra critical of those links and their related
+content.  I'll respect any decisions you make regarding them -->{% endcomment %}
+
+- **Discussion of HTLC mining incentives:** Itay Tsabary, Matan
+  Yechieli, Ittay Eyal [posted][tye post] to the Bitcoin-Dev mailing
+  list with a paper they've written that claims rational miners should
+  want to run a modified Bitcoin node that allows a user to bribe it
+  with a transaction that's timelocked and can't be confirmed until some
+  point in the future.  As a result of this bribe, the miner won't
+  confirm any transaction that could be mined immediately but which
+  conflicts with the bribe (as long as the bribe pays a sufficiently
+  higher feerate than any transactions it blocks).
+
+    If this theory is accurate, it affects Hash TimeLock Contracts
+    (HTLCs) which can be settled immediately by one party (Alice) using
+    a preimage, or which can be settled later by a second party (Bob)
+    after a timeout.  According to the authors, if Bob sees Alice use
+    the preimage to spend the HTLC, Bob should send his conflicting
+    timeout settlement transaction to miners with a sufficiently higher
+    feerate than Alice's preimage settlement---even though miners can't
+    immediately include Bob's transaction---bribing them to ignore
+    Alice's transaction in favor of waiting to confirm his alternative
+    transaction.  This allows Bob to steal the money Alice should
+    receive.  HTLCs are currently used in LN, atomic swaps, and several
+    other contract protocols.
+
+    The authors of the paper propose a solution they call Mutually
+    Assured Destruction HTLCs (MAD-HTLCs).  These require Bob to provide a
+    hashlocked bond that makes him reveal his own preimage when sending
+    his timeout settlement.  If both Alice and Bob reveal their
+    respective preimages, miners will be able to claim both the
+    payment/refund amount and the bond collateral---destroying the
+    incentive for Bob to attempt stealing by bribing miners.
+
+    Downsides to this approach are that it requires Bob to use more
+    collateral---raising the cost of using HTLCs---and it uses more
+    block chain space when MAD-HTLCs are settled onchain compared to
```

Comment:
(I know it's in the style guide but do find that "onchain" looks like a misspelled version of on-chain)

Reply:
I have no strong preference either way.  If you want us to change it, feel free to open a PR updating the style guide.